src/components/ProfileCard/index.tsx

```diff
@@ -114,19 +122,23 @@ export function Ambassadors(): JSX.Element {
       ></ProfileCard>
       <ProfileCard
         name='John Hill'
+        languages="English"
```

Comment:
why sometimes single and sometimes double quote?

Reply:
cause you wont let me use prettier. I never ever look at single v double. Prettier has done that for me for years

Reply:
Feel free to use whatever tools make you happy!